src/test/regress/sql/upgrade_post_11_before.sql

```diff
@@ -129,6 +142,68 @@ END;
 $$;');
 
 CREATE TYPE post_11_upgrade.my_type AS (a int);
+CREATE VIEW post_11_upgrade.view_for_upgrade_test AS SELECT * FROM sensors;
```

Comment:
Once you add support for basic views like `CREATE VIEW basic_view AS SELECT 1` could you also please add a related test as well?

Reply:
added